client/blocks/stripe-utils/normalize.js

```diff
@@ -0,0 +1,256 @@
+/**
+ * External dependencies
+ */
+import { getSetting } from '@woocommerce/settings';
+
+/**
+ * @typedef {import('./type-defs').StripePaymentItem} StripePaymentItem
+ * @typedef {import('./type-defs').StripeShippingOption} StripeShippingOption
+ * @typedef {import('./type-defs').StripeShippingAddress} StripeShippingAddress
+ * @typedef {import('./type-defs').StripePaymentResponse} StripePaymentResponse
+ * @typedef {import('@woocommerce/type-defs/registered-payment-method-props').PreparedCartTotalItem} CartTotalItem
+ * @typedef {import('@woocommerce/type-defs/cart').CartShippingOption} CartShippingOption
+ * @typedef {import('@woocommerce/type-defs/shipping').ShippingAddress} CartShippingAddress
+ * @typedef {import('@woocommerce/type-defs/billing').BillingData} CartBillingAddress
+ */
+
+/**
+ * Normalizes incoming cart total items for use as a displayItems with the
+ * Stripe api.
+ *
+ * @param {CartTotalItem[]} cartTotalItems CartTotalItems to normalize
+ * @param {boolean}         pending        Whether to mark items as pending or
+ *                                         not
+ *
+ * @return {StripePaymentItem[]} An array of PaymentItems
+ */
+const normalizeLineItems = ( cartTotalItems, pending = false ) => {
+	return cartTotalItems
+		.map( ( cartTotalItem ) => {
+			return cartTotalItem.value
+				? {
+						amount: cartTotalItem.value,
+						label: cartTotalItem.label,
+						pending,
+				  }
+				: false;
+		} )
+		.filter( Boolean );
```

Comment:
If you want, you can add the [same change you suggested](https://github.com/Automattic/woocommerce-payments/pull/1529/commits/ffc63af4e91698a542bedd1cbe544039d529bb97) here.

Reply:
Thanks! Fixed in def5d22a6fd469dbb35746cf5b64fdeae61f2e32